.gitignore

```diff
@@ -68,3 +67,36 @@ iOS/nativelibs/arm64
 iOS/nativelibs/i386
 iOS/nativelibs/x86_64
 Solutions/Engine.App/test.ini
+
+libsrc/*.tar*
+libsrc/*.zip
+
+ac2game.dat
+audio.vox
+speech.vox
```

Comment:
Oh, they still use "ac2game.dat" as game file name?

I recall we had a big misunderstanding in the past, when Jochen (@jjsat) just made his ports, he used this as default file name for ags games. Later I had to do a research to find out the origins of this name, and I learnt that this is a very old convention. I think he made a mistake by choosing it, because it is no longer used anywhere in modern code, nor as a compilation output.

Current convention is to call game file as "GameName.ags". The 3.4.0 Editor also produces this file when configured to deploy game for Linux.